daemons/gptp/common/avbts_message.hpp

```diff
@@ -622,7 +655,7 @@ class FollowUpTLV {
 	int32_t cumulativeScaledRateOffset;
 	uint16_t gmTimeBaseIndicator;
 	scaledNs scaledLastGmPhaseChange;
-	int32_t scaledLastGmFreqChange;
+    int32_t scaledLastGmFreqChange;
```

Comment:
Yes, my editor defaults to spaces instead of tabs and I apparently touched this line. Im putting a tab here instead.